doc/source/building/redistributable_binaries.rst

```diff
@@ -10,12 +14,90 @@ artifacts" <https://pypackaging-native.github.io/meta-topics/build_steps_concept
 
 A wheel like that is therefore an intermediate stage to producing a binary that
 can be distributed. That final binary may be a wheel - in that case, run
-``auditwheel`` (Linux), ``delocate`` (macOS) or ``delvewheel`` (Windows) to
-vendor the required shared libraries into the wheel.
+``auditwheel`` (Linux), ``delocate`` (macOS), ``delvewheel`` (Windows) or
+``repairwheel`` (platform-independent) to vendor the required shared libraries
+into the wheel.
 
 The final binary may also be in another packaging format (e.g., a ``.rpm``,
 ``.deb`` or ``.conda`` package). In that case, there are packaging
 ecosystem-specific tools to first install the wheel into a staging area, then
 making the extension modules in that install location relocatable (e.g., by
 rewriting RPATHs), and then repackaging it into the final package format.
 
+
+Build and runtime dependencies
+------------------------------
+
+The Python build and runtime dependencies that are needed to build SciPy can
+be found in the ``pyproject.toml`` metadata. Note that for released versions of
+SciPy, dependencies will likely have upper bounds. Each upper bound has
+comments above it; packagers are free to remove or loosen those upper bound in
+most cases (except for ``numpy``). E.g.::
+
+    # The upper bound on pybind11 is pre-emptive only
+    "pybind11>=2.12.0,<2.13.0",
+
+    #   ...
+    #   3. The <2.3 upper bound is for matching the numpy deprecation policy,
+    #      it should not be loosened.
+    "numpy>=2.0.0rc1,<2.3",
+
+Non-Python build requirements are:
+
+- C, C++ and Fortran compilers
+- BLAS and LAPACK libraries
+- ``ninja``
+- ``pkg-config``
```

Comment:
Can you get away with the CMake equivalent? I thought I saw the config stage checking for that at one point when I didn't have `pkg-config` for finding the linalg backend, but yeah I always use `pkg-config` now I think.

Reply:
You can in principle, but it's not well-tested so I would not want to go into that here.